lib/bank_api/clients/banco_de_chile_company_client.rb

```diff
@@ -75,9 +75,19 @@ def goto_deposits
 
     def select_deposits_range
       browser.search('input[name=initDate]').set(deposit_range[:start])
+      first_account = browser.search("select[name=ctaCorriente] option").find do |account|
+        account.value != "TODAS"
+      end.value
+      browser.search("select[name=ctaCorriente]").set by_value: first_account
       browser.search('#consultar').click
     end
 
+    def validate_banchileqlo_status!
+      unless browser.search(".textoerror:contains('no podemos atenderle')").none?
+        raise "Banchileqlo is down"
```

Comment:
Esta bien hablar así en slack, pero no el código. 

`Don’t get cute with naming by means of jokes (inside or well known)`

https://www.codingblocks.net/podcast/clean-code-writing-meaningful-names/